core/src/main/java/org/infinispan/stream/impl/AbstractCacheStream.java

```diff
@@ -284,7 +285,8 @@ public void close() {
             if (localRun) {
                localValue = op.performOperation();
                // TODO: we can do this more efficiently - since we drop all results locally
-               if (dm.getReadConsistentHash().equals(ch)) {
+               LocalizedCacheTopology newCacheTopology = dm.getCacheTopology();
```

Comment:
Just to confirm this is just to trace the cache topology here right? There is no functional difference?

Reply:
Yes, initially I thought the segments were marked as lost because of a topology change, so I wanted to log the topology ids.